js-green-licenses.json

```diff
@@ -4,6 +4,7 @@
 		"GPL-2.0",
 		"GPL-2.0-or-later",
 		"GPL-2.0+",
+		"GPL-3.0",
```

Comment:
Interesting, I thought that `GPL-3.0` is included in `"GPL-2.0-or-later"`, in the `or-later` part :)

Reply:
ya I thought the exact same thing, but 2+ must mean things like 2.1, 2.1, etc